masonry/src/doc/06_masonry_concepts.md

```diff
@@ -100,6 +100,26 @@ A widget is considered "interactive" if it can still get text and/or pointer eve
 Stashed and disabled widget are non-interactive.
 
 
+## Bounding rect
+
+A widget's bounding rect is a window-space axis-aligned rectangle inside of which pointer events might affect either the widget or its descendants.
+
+In general, the bounding rect is a union or a widget's layout rect and the bounding rects of all its descendants.
+
+The bounding rects of the widget tree form a kind of "bounding volume hierarchy": when looking to find which widget a pointer is on, Masonry will automatically exclude any widget if the pointer is outside its bounding rect.
+
+<!-- TODO - Include illustration. -->
+
+<!-- TODO - Add section about clip paths and pointer detection. -->
```

Comment:
Do you plan to handle the TODOs in this PR?

Reply:
We need to refactor how clip paths work at some point, it would be part of that.